src/rv/ui/screens/ViewerScreenBase.java

```diff
@@ -34,9 +34,9 @@
 import rv.world.WorldModel;
 import rv.world.objects.Agent;
 
-public abstract class ViewerScreenBase extends ScreenBase
-        implements KeyListener, MouseListener, MouseMotionListener, MouseWheelListener,
-        GameState.GameStateChangeListener, WorldModel.SelectionChangeListener {
+public abstract class ViewerScreenBase extends ScreenBase implements KeyListener, MouseListener,
```

Comment:
Yeah, I noticed that after formatting there were changes to parts of the code I hadn't touched.